docs/main/modules/getting-started/pages/index.adoc

```diff
@@ -5,7 +5,7 @@ This document will guide you through the fundamentals of the Apache Camel Core.
 [[BookGettingStarted-eip-book]]
 
 [[BookGettingStarted-TheEnterpriseIntegrationPatternsEIPBook]]
-== The _Enterprise Integration Patterns_ (EIP) book
+== The _Enterprise Integration Patterns_ (E.I.P.) book
```

Comment:
We use EIP as the reference name, and not with dots

Reply:
So, Grammarly complains about using it this way. I wonder if we should reduce usage of the term, as this  indicates it affects readability.

Any thoughts about this?

Reply:
Btw, I'll merge this for now and we can think about this later